web-packages/imgui-demo/src/panels/node-graph/tools-graph/nodes/flow-tools-node.ts

```
@@ -0,0 +1,411 @@
+export abstract class FlowToolsNode extends NodeGraph.FlowNode {
+  static GetPinTypeForValueType (valueType: GraphValueType): string {
+    return GetIDForValueType(valueType);
+  }
+
+  static GetValueTypeForPinType (pinType: string): GraphValueType {
+    return GetValueTypeForID(pinType);
+  }
+
+  constructor () {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)